sources/assets/Xenko.Core.Packages/ProgressReport.cs

```diff
@@ -10,7 +10,7 @@ namespace Xenko.Core.Packages
     public class ProgressReport : IDisposable
     {
         private readonly NugetStore store;
-        private readonly string version;
+        //private readonly string version;
```

Comment:
Remove, don't comment out.

Reply:
OK as is, let's delete all related code in a separate pass